src/items/class-features/dark_matter_senses_(su).json

```diff
@@ -0,0 +1,115 @@
+{
+  "_id": "qIddPtbrsYRB8DBV",
+  "name": "Dark Matter Senses (Su)",
+  "type": "feat",
+  "img": "systems/sfrpg/images/cup/gameplay/graviton.webp",
+  "system": {
+    "type": "",
+    "ability": null,
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "move",
+      "condition": "",
+      "cost": 1
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "none",
+      "value": null
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.G7p2YIGFTAU6uGjL]{Stellar Revelation}</p>\n<p>@Icon[type:graviton]<sub>&nbsp;</sub>Graviton Revelation</p>\n<p>Your sensitivity to dark matter allows you to navigate in lightless conditions. You gain @UUID[Compendium.sfrpg.racial-features.HhrIJeHTxSGVX9mV]{Darkvision} with a range of 60 feet.</p>"
```

Comment:
Don't link to  a specific species' species feature. Link to the journal describing darkvision

Reply:
Can you provide some pointers on how to find the correct journal?

Reply:
Off the top of my head, I believe Darkvision is under the ability descriptions journal in the SFRPG rules compendium. Or maybe in the senses journal if that exists?